packages/react-native/src/index.ts

```diff
@@ -10,15 +10,21 @@ class Honeybadger extends Client {
   private __nativeHandlerInitialized:boolean
   private __originalJsHandler:(error: Error, isFatal: boolean) => void
 
+  protected override __notifier = {
+    name: '@honeybadger-io/react-native',
+    url: 'https://github.com/honeybadger-io/honeybadger-js/tree/master/packages/react-native',
+    version: '__VERSION__'
```

Comment:
Need to add a rollup plugin for this to work, no? 

Also the `react-native` package currently sets a notifier within `transport.ts` -- that code can now be removed, correct?

Reply:
Yes, nice catch!

Reply:
Done!